### ISSUE TITLE
Multiple apps with same functionality on Cinnamon

### ISSUE THREAD

**Issue by DarkEra** [kororaproject/kp#94]
On our Cinnamon spin we have multiple apps that provide the same function. By example: Shotwell let's users view and organize photo's/pictures, Pix that is included is a part of Xapps (from Mint) which ships by default on Cinnamon, provides the same funcionality: it lets users view and organize images.
Media Player (a.k.a. Xplayer from Mint's Xapps) lets users watch movies, so does VLC.
So we need to talk about what we want to keep in our Cinnamon spin and what can be dropped. Are we going to keep the Xapps and drop Shotwell and VLC or keep Shotwell and VLC but drop a few of the Xapps?

I also noticed that Rhythmbox and Sound Juicer are included, the latter is used to rip CD's however... Rhythmbox can rip CD's too. So there's no use in keeping Sound Juicer imo when Rhythmbox provides that feature.
The two are also present on our Gnome spin.

**Comment by AdmiralAsshat** [comment]
I agree that removing extraneous apps with duplicated functionality is a good idea and would go some ways towards improving the perception that Korora is a "bloated" distro.

With that said, I have a certain fondness for VLC, so if we consider removing it in favor of Xplayer, I'd like to know the reasoning, namely:

1. I assume Korora's philosophy is to follow "upstream". Does that mean upstream Cinnamon or upstream Fedora? VLC is in base Fedora, if memory serves.
 
2. When does Korora deviate from upstream? If there's a "gap" as far as a program not provided, that seems like an obvious case. However, if the Korora team considers a variant app far superior to an upstream standard app, will Korora deviate for the sake of including the better app to the end-user? Case in point, the version of Xplayer I just installed doesn't seem to integrate with the MPRIS service, so I don't get notifications in the sound tray. I do in VLC, which makes it more attractive to the end-user, in my eyes. VLC also seems to have better built-in support for playing online links like Youtube, etc.

3. If we decide to remove App B because App A always has that feature, does the feature need to be sufficiently robust or accessible in order to render App B obsolete? For instance, I see that Audacious' option to rip (it's a bit buried, and I couldn't test it as my laptop doesn't have a CD tray, but it is at least there and not buried behind an option plug-in or something), but [this article](https://www.maketecheasier.com/rip-audio-cd-in-linux/) seems to indicate that the ripping process is much slower than Sound Juicer, whose sole purpose is to rip CD's. I guess to loop back to Korora's outlook, do we want to favor broad multi-taskers for the sake of fewer apps, or do we want UNIX-philosophy utilities that do one thing and do it well?

**Comment by jimkp** [comment]
> With that said, I have a certain fondness for VLC, so if we consider removing it in favor of Xplayer, I'd like to know the reasoning, namely:

As VLC is standard right across all the desktops I don't expect we would remove it. It is included in addition to other applications is the other desktops as there isn't a direct comparison to it.

**Comment by jimkp** [comment]
I'm going to consider this as a Cinnamon thread and so add general comments for Cinnamon 27 here.

Increase the icon size in Plank as it is considerably smaller than the panels on other desktop. Also fix the order of launchers to match the other desktops.
Replace the default Fedora background on the login screen with the Korora background.

**Comment by DarkEra** [comment]
Increase the size of Plank from 32 to 48.

**Comment by DarkEra** [comment]
Disable sound effects on Cinnamon.

**Comment by DarkEra** [comment]
Added numlockx to the template so it's included now.

**Comment by jimkp** [comment]
add terminal to Plank

I think there is a new terminal application which is how it has gone missing
